ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/OptimizeService.java

```diff
@@ -509,4 +494,22 @@ private List<TableOptimizeRuntime> selectTableOptimizeRuntimes() {
       return tableOptimizeRuntimeMapper.selectTableOptimizeRuntimes();
     }
   }
+
+  private void scheduleRefresh() {
+    LOG.info("start schedule refresh");
+    new Thread(() -> {
```

Comment:
It's not a good practice to call `new Thread`, we'd better create the thread in `ThreadPool` like the other threads in AMS.